site/src/pages/WorkspacesPage/filter/menus.ts

```diff
@@ -32,7 +33,8 @@ export const useTemplateFilterMenu = ({
       return null;
     },
     getOptions: async (query) => {
-      const templates = await getTemplates(orgId);
+      // Show all templates including deprecated
+      const templates = await getTemplates(orgId, undefined);
```

Comment:
Just adding an additional point for my previous comment, even the interface would be easier to understand:
```ts
getTemplates(orgId, { deprecated: false  })
```